lib/ws-kernel-picker.js

```diff
@@ -91,7 +97,9 @@ export default class WSKernelPicker {
           this.onSession.bind(this)
         );
         sessionListing.previouslyFocusedElement = this.previouslyFocusedElement;
-        sessionListing.setLoading("Loading sessions...");
+        sessionListing.selectListView.update({
+          loadingMessage: "Loading sessions..."
+        });
```

Comment:
I'll do some more experimenting later. I tried wrapping calling `render` and wrapping these lines in their own `then` call after the current one. Didn't try using the debug tools though, that's a good idea too.